test/services/system.spec.js

```diff
@@ -0,0 +1,40 @@
+import { getSystemData } from '../../services'
+
+describe('Get system data from external source', () => {
+  test('it returns a valid system data', async () => {
```

Comment:
Esta linha tem uma sacada interessante a respeito do uso do Jest.
Ao invés de `test('it returns ...` -- que fica pouco legível, imagina a frase "test it returns a valid" -- o Jest permite que você use a função `it`, que fica `it('returns ...`